### PR TITLE
Delta: Nanite lab, tweaks/fixes

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Added a nanite lab to Deltastation! It's at the old EXPERIMENTOR lab.
tweak: Delta: Moved the Xenobiology disposals bin to be less obstructive. Added two sets of insulated gloves to Engineering.
fix: Delta: Fixed scientists not having maintenance access near the circuitry lab and toxins launch chamber.
/:cl:

I downsized Delta's EXPERIMENTOR lab and used the free space to add a nanite lab - two workspaces since Delta is for highpop.

Other than that, I added two sets of insulated gloves to Engineering; people were complaining that engineers are running out of gloves when the department is fully staffed. 
The Xenobio disposals bin is now in a less obstructive area and scientists have access to the maint doors near circuitry/toxins launch site.

nanites son
closes #40089

### CODE REVIEW

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -81858,66 +81854,53 @@
...
 /turf/open/floor/plasteel/vault{
 	dir = 5
 	},
-/area/science/explab)
+/area/science/nanite)
 "duD" = (
-/obj/machinery/atmospherics/components/unary/vent_pump/on{
-	dir = 8
+/obj/structure/chair/office/light{
+	icon_state = "officechair_white";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: dv

if you wouldn't mind removing this from Delta, Box, Pubby, and Emergency_Cere, that'd be appreciated.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 1 above)
Comment (81Denton) [reply _maps/map_files/Deltastation/DeltaStation2.dmm]: O yeah, shouldn't take too long.

**Comment by GunHog** [comment]
>Putting the nanite machines next to the EMP generator

This is going to get interesting!

**Comment by 81Denton** [comment]
@Cyberboss Mapdiffbot isn't picking up this PR, should I close and open a new one?

**Comment by Cyberboss** [comment]
Mdb will be experiencing technical difficulties for a while

**Comment by 81Denton** [comment]
I was able to enter it when I first tested it, but let me check if it actually works.

**Comment by 81Denton** [comment]
@coldud13 Just tested it locally, both work.

**Comment by XDTM** [comment]
last i've heard it's just an issue with windows

**Pull request merged**